exercises/G-conditionals-2/exercise-3.js

```diff
@@ -8,7 +8,16 @@
   - Otherwise the grade is "F"
 */
 
-function calculateGrade(mark) {}
+function calculateGrade(mark) {
+  if (mark >= 80 ){
+    return "A";
+  }else if (mark <= 80 && mark >= 60 ){
```

Comment:
the `mark <=80` check is unecessary here, as anything `mark >=80` will be caught by the above `if` statement